src/TestFramework/Microsoft.Rest.ClientRuntime.Azure.TestFramework/TestEnvironmentFactory.cs

```diff
@@ -26,307 +27,6 @@ public static class TestEnvironmentFactory
         /// Sample Value 2 - Prompt for login credentials:
         /// TEST_CSM_ORGID_AUTHENTICATION=SubscriptionId={subscription-id};AADAuthEndpoint=https://login.windows-ppe.net/;BaseUri=https://api-next.resources.windows-int.net/
         /// </summary>
-        const string TestCSMOrgIdConnectionStringKey = "TEST_CSM_ORGID_AUTHENTICATION";
```

Comment:
Moving all this logic out of TestEnvironmentFactory makes it somewhat difficult to review this PR, as it's difficult to see what has changed and if the same scenarios are covered.  Also, it is odd to make blocking calls inside a constructor, which is one reason all the authentication happened in the factory method.  Unless there is a really compelling reason to put this into the constructor, this logic should be in the factory.